pilot/pkg/security/authn/v1beta1/policy_applier_test.go

```diff
@@ -486,14 +486,15 @@ func TestJwtFilter(t *testing.T) {
 			},
 		},
 		{
-			name: "JWT policy with bad Jwks URI",
```

Comment:
Changed the position of `bad jwks uri` test because it initiates the background fetch and since we are trying to fetch `pubkey` for all the jwksuri in the cache we would update the key for other jwksuri (as our mock server returns pubkey1 in first hit and pubkey2 for rest of the hits). If we don't do this, it would create some flakiness in the code.